shared-bindings/wifi/Radio.c

```diff
@@ -149,7 +149,7 @@ const mp_obj_property_t wifi_radio_mac_address_obj = {
                MP_ROM_NONE },
 };
 
-//|     mac_address_ap: ReadableBuffer
+//|     mac_address_ap: str
```

Comment:
`ReadableBuffer` is correct here. It's not a human-readable string, it's the bytes of the MAC address.
```suggestion
//|     mac_address_ap: ReadableBuffer
```